### PR TITLE
add Kalamazoo template

### PR DESCRIPTION
**Pull request by Ceshion**
added the template for Kalamazoo using DC template as an example, is the language all right?

### CODE REVIEW

File: _emails/kalamazooMi.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
+---
+title: Kalamazoo, MI
+permalink: "/kalamazooMi"
```
Comment (teddywilson) [inline _emails/kalamazooMi.md:3]: Let's just leave this at /kalamazoo for now?

File: _emails/kalamazooMi.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+- Jennifer.Aniano@kalcounty.com
+- Roger.Tuinier@kalcounty.com
+- John.Gisler@kalcounty.com
+- Christine.Morse@kalcounty.com
+- Mike.Quinn@kalcounty.com
+- Meredith.Place@kalcounty.com
+- SenSMcCann@senate.michigan.gov
+- jonhoadley@house.mi.gov
+subject: Regarding the funding of KDPS
+body: "Hello, \n\nI am a constituent living in Kalamazoo County and I am writing to you
```
Comment (teddywilson) [inline _emails/kalamazooMi.md:29]: make all of the fields here in brackets  all caps - e.g., [YOUR NAME]

File: _emails/kalamazooMi.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+School officials have been sounding the alarm that due to COVID-19 tax shortfalls, they could be
+facing unprecedented cuts to per pupil funding. Yet KDPS is accepting applications for new 
+officers despite already having more employees than any other city department. Public health 
+workers in Kalamazoo have had to sew their own face masks to serve consumers safely during the 
+pandemic. Yet KDPS has a stockpile of face shields they are using for public intimidation, not 
+protection. Kalamazoo Public Safety takes up 48% of the entire general fund budget for 2020. Other 
+public services account for a paltry 8% and parks and recreation a depressing 4% of budget. 
+That is outrageous and needs to change. Kalamazoo needs dollars for services that actually 
+improve and lift up our community. Dollars for education, for food assistance, for housing, 
+for people. Not policing.\n\nSincerely,\n[your name]\n"
```
Comment (teddywilson) [inline _emails/kalamazooMi.md:52]: Let's include name, email, phone number, address

apparently emails without this are being filtered out.

Comment (Ceshion) [reply _emails/kalamazooMi.md:52]: Understood, the edits will be in momentarily

File: _emails/kalamazooMi.md (hunk 2 above)
Comment (Ceshion) [reply _emails/kalamazooMi.md:29]: I'll assume you mean in general, these lines don't have any bracketed statements

File: _emails/kalamazooMi.md (hunk 1 above)
Comment (Ceshion) [reply _emails/kalamazooMi.md:3]: what should be the convention for more common city names if they appear?

**Pull request merged**